Modules/getpath.c

```diff
@@ -1297,15 +1297,15 @@ calculate_zip_path(PyCalculatePath *calculate)
     PyStatus res;
 
     /* Path: <PLATLIBDIR> / "python00.zip" */
-    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python000.zip");
+    wchar_t *path = joinpath2(calculate->platlibdir_macro, L"python" Py_STRINGIFY(PY_MAJOR_VERSION) Py_STRINGIFY(PY_MINOR_VERSION) L".zip");
     if (path == NULL) {
         return _PyStatus_NO_MEMORY();
     }
 
     if (calculate->prefix_found > 0) {
         /* Use the reduced prefix returned by Py_GetPrefix()
 
-           Path: <basename(basename(prefix))> / <PLATLIBDIR> / "python000.zip" */
+           Path: <basename(basename(prefix))> / <PLATLIBDIR> / ZIPNAME */
```

Comment:
ZIPNAME is not defined, I suggest:

```suggestion
           Path: <basename(basename(prefix))> / <PLATLIBDIR> / "pythonXY.zip" */
```